codespell_lib/data/dictionary.txt

```diff
@@ -12787,6 +12787,8 @@ plattforms->platforms
 plausability->plausibility
 plausable->plausible
 playble->playable
+playge->plague
+playgerise->plagiarize
```

Comment:
Given this has ise, it should correct to ise, but maybe add the ize variant too.

Reply:
Done

Reply:
I meant playgerise->plagiarise, you've added playgerize->plagiarize but not changed the original one.

Reply:
Done